openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -997,7 +1012,11 @@ port_INLINE void activity_ti2() {
    packetfunctions_reserveFooterSize(&ieee154e_vars.localCopyForTransmission, 2);
    
    // calculate the frequency to transmit on
-   ieee154e_vars.freq = calculateFrequency(schedule_getChannelOffset()); 
+   if (ieee154e_vars.isSpouriousEB == TRUE) {
+      ieee154e_vars.freq = calculateFrequency(SCHEDULE_MINIMAL_6TISCH_CHANNELOFFSET); 

```

Comment:
This is a great idea. Please:
- declare a new variable `EB_CHANNELOFFSET`, set to `15`
- return an error when a module tries to add a cell on channelOffset equal to `EB_CHANNELOFFSET`.

This will avoid collisions between EB and other dedicated traffic.